packages/connect/src/data/DataManager.ts

```
@@ -46,7 +48,12 @@ export class DataManager {
             const modelType = DeviceModelInternal[model as keyof typeof DeviceModelInternal];
             // Check if the firmware data exists for this model
             if (this.assets[firmwareKey]) {
-                parseFirmware(this.assets[firmwareKey], modelType);
+                fillFirmwareReleases(this.assets[firmwareKey], modelType);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)